PubNub/Core/PubNub+Publish.m

```diff
@@ -259,6 +267,7 @@ - (void)sizeOfMessage:(id)message toChannel:(NSString *)channel compressed:(BOOL
             }
             PNRequestParameters *parameters = [self requestParametersForMessage:messageForPublish
                                                                       toChannel:channel
+                                                                   withMetadata:nil
```

Comment:
@jzucker2 We need few more methods to correctly measure payload size taking into account _meta_ in query string.